frontend/interfaces/team.ts

```diff
@@ -8,10 +9,32 @@ export default PropTypes.shape({
   role: PropTypes.string,
 });
 
+/**
+ * The shape of a team entity
+ */
 export interface ITeam {
   name: string;
   id: number;
-  hosts: number;
-  members: number;
+  host_count: number;
+  user_count: number | IUser[];
```

Comment:
I find it a little confusing that these inconsistent. Is this going to be updated in the hosts work?

Reply:
yah not sure why i had it being number or list of users. removed list of users type.